docs/QSG/build_from_scratch.md

```diff
@@ -92,7 +93,7 @@ Once all prerequisites installed and the sources are downloaded,
 DAOS can be built via the following command:
 
 ```bash
-$ scons --config=force --build-deps=yes install
+$ scons -j 4 --config=force --build-deps=yes install
```

Comment:
Would it be better to use `-j$(nproc)` instead of `-j 4`?

Reply:
Then you need to define what nproc is.

I think using the long-form in documentation and reference code is good in general and in this case it would also make it clear to the user that they were able to pick a value whereas with -j then they're more likely to just use it without thinking so I'd prefer --jobs here.

There's also a --build-deps=only option now that we should probably use here, it wlll build the deps but not daos so an extra step is needed afterwards but again, this makes it easier to use as the last step can then be replayed on source changes without having to rebuild everything.

Reply:
That's a nice feature of `make` but `scons` does not support that.

Reply:
I see. `--jobs $(nproc)` is the best as it gives the most efficient compilation.
Additionally, it does not suggest any particular value to be used.

Reply:
I had misread that command as requiring the user to set an environment variable called nprocs, I see now that that's calling a subcommand and that subcommand is in coreutils so should always be present.  Yes, I agree that long-form --jobs with $(nproc) makes the most sense here.

Reply:
Done. Thanks for the comments.